### ISSUE TITLE
how to run functions

### ISSUE THREAD

**Issue by Palashio** [JielongZ/3D-UNet-PyTorch-Implementation#3]
so i've got a set of 3D images (128, 128, 128) (both images and their appropriate masks). I want to run this implementation of UNet on it. How would I do this? They're `.nii` files so i've coded up my on dataset loader for it. I'm a bit confused on where to get started as there aren't enough reference resources.

**Comment by Palashio** [comment]
I have it working so that this is what my training script looks like so far:

```
    for epoch in range(epochs):
        print(' - training - ')
        for i, (images, masks) in enumerate(train_loader):
            images = images.to(device)
            masks = masks.to(device)
```

where images, masks are both sizes (1, 1, 128, 128, 128) and train_loader looks like this: 

```
train_loader = DataLoader(dataset=Dataset(partition['orig'], partition['segment']), 
                          batch_size = batch_size, shuffle = True)
```

where partition['orig] is the original image and partition['segment'] is the segmented image. the batch size is also 1.

**Comment by zyl200846** [comment]
i will update info this weekend so that you can run the code successfully. Its been busy since i joined the job. sorry for causing inconvenience.

best regards，

Jielong

Thanks & regards,
Jielong a Hong
________________________________

**Comment by NadiyaC249** [comment]
why  i got  a vary poor  performance on the model ？（use the brats dataset  to segment brain  tumor）
